### PR TITLE
docs: reorganise the readme

### PR DESCRIPTION
**Pull request by WilcoFiers**
One of the most common questions people have is what rules axe-core has, and how they map to WCAG. I hope that by being more explicit in the readme, and putting this section higher up we can limit how often this question is asked.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,24 +11,12 @@ Axe is an accessibility testing engine for websites and other HTML-based user in
...
+Axe-core has different types of rules, for WCAG 2.0 and 2.1 on level A and AA, as well as a number of best practices that help you identify common accessibility practices like ensuring every page has an `h1` heading, and to help you avoid "gotchas" in ARIA like where an ARIA attribute you used will get ignored.
 
-Axe was built to reflect how web development actually works. It works with all modern browsers, tools, and testing environments a dev team might use. With axe, accessibility testing can be performed as part of your unit testing, integration testing, browser testing, and any other functional testing your team already performs on a day-to-day basis. Building accessibility testing into the early development process saves time, resources, and all kinds of frustration.
+The complete list of rules, grouped WCAG level and best practice, can found in [doc/rule-descriptions.md](./doc/rule-descriptions.md).
```
Comment (straker) [inline README.md:18]: Another common question we get is about coverage of the rules and if axe can be used to be compliant or ADA maturity. I think it'd be worthwhile to say that the rules don't map exactly to each WCAG success criterion completely as no WCAG criterion can be fully automated.

**Review by straker — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -11,24 +11,12 @@ Axe is an accessibility testing engine for websites and other HTML-based user in
 
 [Sign up for axe news](https://hubs.ly/H0fsN0b0) to get the latest on axe features, future releases, and events.
 
-## Philosophy
```
Comment (dylanb) [inline README.md:14]: Why are we removing all of the philosophy stuff?

File: README.md [hunk 3]
```diff
@@ -11,24 +11,12 @@ Axe is an accessibility testing engine for websites and other HTML-based user in
...
-
-The web can only become an accessible, inclusive space if developers are empowered to take responsibility for accessibility testing and accessible coding practices.
+## The Accessibility Rules
 
-Automated accessibility testing is a huge timesaver, it doesn't require special expertise, and it allows teams to focus expert resources on the accessibility issues that really need them. Unfortunately, most accessibility tools are meant to be run on sites and applications that have reached the end of the development process and often don't give clear or consistent results, causing frustration and delays just when you thought your product was ready to ship.
+Axe-core has different types of rules, for WCAG 2.0 and 2.1 on level A and AA, as well as a number of best practices that help you identify common accessibility practices like ensuring every page has an `h1` heading, and to help you avoid "gotchas" in ARIA like where an ARIA attribute you used will get ignored. With axe-core, you can find **up to 50% of WCAG issues automatically**.
```
Comment (dylanb) [inline README.md:16]: Add a mention of the needs review items?

File: README.md (hunk 1 above)
Comment (dylanb) [reply README.md:18]: Perhaps we could point to the axe extension to give people an option for helping with the stuff that is not 100% automatable.

**Review by straker — APPROVED** [review]

**Pull request merged**